app/display/model/src/main/resources/display_model_preferences.properties

```diff
@@ -32,7 +32,7 @@ font_files=examples:font.def
 # * May then contain characters or numbers
 # * May also contain underscores
 # 
-macros=<EXAMPLE_MACRO>Value from Preferences</EXAMPLE_MACRO><TEST>true</TEST>
```

Comment:
Can we keep an example in here? User needs to know if the syntax is `EXAMPLE_MACRO=Xyz` or `<EXAMPLE_MACRO>Xyz</EXAMPLE_MACRO`.

How about this:
```
#macros=<EXAMPLE_MACRO>Value from Preferences</EXAMPLE_MACRO><TEST>true</TEST>
macros=
```

```

Reply:
Removing LCID is good, and making the default for macros nothing instead of that EXAMPLE_MACRO and TEST is also good, just keep the example as a comment